sdk/core/azure-core/src/test/java/com/azure/core/http/policy/RequestIdPolicyTests.java

```diff
@@ -108,7 +108,7 @@ public void sameRequestIdForRetry() throws Exception {
                     return Mono.just(mockResponse);
                 }
             })
-            .policies(new RequestIdPolicy(), new RetryPolicy(new RetryPolicyOptions(new FixedDelay(1, Duration.of(0, ChronoUnit.SECONDS)))))
+            .policies(new RequestIdPolicy(), new RetryPolicy(new FixedDelay(1, Duration.of(0, ChronoUnit.SECONDS))))
```

Comment:
Do you have a test for customized request-id per request?

Reply:
I will add one.